### DISCUSSION TITLE
Change color of line break symbol (pilcrow)

### DISCUSSION THREAD

**Discussion by u-fischer** [WinEdt-Team/WinEdt-Support#68] [1.1 General Help and Discussion]
I like to enable `Show Line Breaks` to see where I have hard line breaks, but the default color is too vivid to my liking. 

Where can I adjust that?

**Comment by Alex-Simonic** [comment]
Adding the following to my Private Settings works for me:

```
[COLORS*]

COLOR_SCHEME="Color Base: Custom"
COLORS="END_LIST"
  fscHRT="#A8A8A8"
END_LIST
END="Color Base: Custom"
```
fscHRT could also be modified directly in Colors.ini...

**Comment by u-fischer** [comment]
Perfect, `fscHRT` was the magic word I couldn't find ;-)
